horace_core/algorithms/read_horace.m

```diff
@@ -25,58 +25,55 @@
 %
 %
 
-%
-
 % Perform operations
 % ------------------
 % Check number of arguments
 n_outputs = nargout;
-if n_outputs>nargin
+if n_outputs > nargin
     error('HORACE:read_horace:invalid_argument',...
         'number of output objects requested is bigger then the number of input files provided')
 end
-%
-[ok,mess,get_dnd,force_sqw,argi] = parse_char_options(varargin,{'-get_dnd','-force_sqw'});
+
+[ok,mess,get_dnd,force_sqw,file_backed,argi] = parse_char_options(varargin,{'-get_dnd','-force_sqw','-file_backed'});
 if ~ok
     error('HORACE:read_horace:invalid_argument',...
         mess);
 end
-%
+
 if get_dnd && force_sqw
     error('HORACE:read_horace:invalid_argument',...
-        'only one option i.e. -get_dnd or -force_sqw can be provided simultaniously');
+        'only one option i.e. -get_dnd or -force_sqw can be provided simultaneously');
+end
+
+if ~any(strcmp(argi, 'file_backed')) % Prioritise explicit, but otherwise respect -file_backed
+    argi = [argi, 'file_backed', file_backed];
```

Comment:
Yes, rephrased. If `file_backed` specified as kwarg (e.g. `'file_backed', false`) it will not override the user, but will attempt to file back in the case of the `-file_backed` flag (which sets the variable `file_backed`).